src/component/sidebaritem/index.tsx

```diff
@@ -55,50 +59,52 @@ function SidebarItem(props: SidebarItemProps) {
       : `/${ventureHandle}/settings`;
 
   return (
-    <PlasmicSidebarItem
-      {...rest}
-      root={{
-        ref: dropdownRootRef,
-      }}
-      isDropdown={dropdownVisible ? "isDropdown" : undefined}
-      isOwner={userRole === "owner" ? "isOwner" : undefined}
-      icon={{
-        onClick: (e) => {
-          e.stopPropagation(); // Prevent root onClick from triggering when collapsing group
+    <div ref={ref}>
```

Comment:
I don't like the wrapper just for the ref, but the alternatives (combining refs) are much more complicated.